openedx/core/djangoapps/theming/helpers.py

```diff
@@ -1,9 +1,15 @@
 """
     Helpers for accessing comprehensive theming related variables.
 """
+import logging
+
+from django.conf import settings
+
 from microsite_configuration import microsite
 from microsite_configuration import page_title_breadcrumbs
-from django.conf import settings
+
+
+log = logging.getLogger(__name__)
```

Comment:
We can remove all the changes above with the latest version of the implementation.